dxr/plugins/tests/test_pygmentize_crlf.py

```diff
@@ -0,0 +1,28 @@
+from nose.tools import ok_
```

Comment:
A nit: we should probably turn pygmentize into a package (folder) and put the tests inside it. See `plugins/descriptor` for a precedent.